front/src/components/ConfirmationModal.svelte

```diff
@@ -13,29 +13,22 @@
   export let committee: CommitteeType = CommitteeType.OPERATIONS;
 
   export let changeStatus = () => {};
-  export let toggleModal = () => {};
+  export let toggleConfirmationModal = () => {};
 
   export let modalText: string = '';
+  export let modalHeadingText: string = '';
 
-  export let committees = [
-    { id: 0, committee: CommitteeType.OPERATIONS },
-    { id: 1, committee: CommitteeType.DEVELOPMENT },
-    { id: 2, committee: CommitteeType.HACKEREXPERIENCE },
-    { id: 3, committee: CommitteeType.DESIGN },
-    { id: 4, committee: CommitteeType.SPONSORSHIP },
-    { id: 5, committee: CommitteeType.CONTENT },
-    { id: 6, committee: CommitteeType.MARKETING }
-  ];
+  export let committees = [];
 </script>
 
 <Modal
   size="sm"
   hasScrollingContent={false}
   bind:open
-  modalHeading="Change applicant status"
+  modalHeading={modalHeadingText}
```

Comment:
let's just name it `modalHeading` and do this instead
```suggestion
  {modalHeading}
```